cinema/views.py

```diff
@@ -1 +1,75 @@
-# write views here
+from rest_framework.viewsets import ModelViewSet
+
+from cinema.models import (
+    Movie,
+    Genre,
+    Actor,
+    CinemaHall,
+    MovieSession
+)
+from cinema.serializers import (
+    MovieSerializer,
+    MovieListSerializer,
+    GenreSerializer,
+    ActorSerializer,
+    CinemaHallSerializer,
+    MovieSessionSerializer,
+    MovieSessionListSerializer,
+    MovieSessionDetailSerializer,
+    MovieDetailSerializer
+)
+
+
+class MovieViewSet(ModelViewSet):
+    queryset = Movie.objects.all()
+    serializer_class = MovieSerializer
+
+    def get_queryset(self):
+        queryset = self.queryset
```

Comment:
seems like this variable is redundant(and it doesn't simplify your code). You can just use `self.queryset`.
e.g.
```python
if self.action in ...:
    return self.queryset....
return self.queryset
```

Reply:
just followed example from conspectus, corrected now